apps/laboratory/src/components/Wagmi/WagmiSignMessageTest.tsx

```diff
@@ -4,7 +4,7 @@ import { useSignMessage } from 'wagmi'
 import { ConstantsUtil } from '../../utils/ConstantsUtil'
 import { useChakraToast } from '../Toast'
 import { useAppKitAccount } from '@reown/appkit/react'
-import type { Address } from 'viem'
+import { type Address } from 'viem'
```

Comment:
Just to know, is there any difference?

Reply:
I added some other imports, seems this was auto-changed